svcdef/svcdef.go

```diff
@@ -373,3 +383,14 @@ func NewField(f *ast.Field) (*Field, error) {
 	}
 	return rv, nil
 }
+
+// isExported returns true if the provided name of a declaration begins with a
+// capital letter.
+func isExported(name string) bool {
+	// Abuse DecodeRuneInString to get the first rune in the string
```

Comment:
Abuse is a pretty colorful word. I don't think it belongs. 

Also how is this an abuse? The standard library does something similar. https://github.com/golang/go/blob/832082b44e2965c136e53a9b8009d2e860766000/src/cmd/compile/internal/gc/export.go#L51